### DISCUSSION TITLE
How create partitions.

### DISCUSSION THREAD

**Discussion by andreysneg** [dbeaver/dbeaver#21739] [Q&A]
Documenation say:

[Creating Partitions](https://dbeaver.com/docs/dbeaver/How-to-work-with-database-Partitions/#creating-partitions)

1) The first step to creating Partitions is to [create a table](https://dbeaver.com/docs/dbeaver/How-to-work-with-database-Partitions/New-Table-Creation). Consider a table with the following columns:

2) Next, go to the Properties tab, and find the Partition by field. Here, you need to specify the Partition criteria. For instance, to Partition the data based on the year column, you can use RANGE (year).

Where is PARTITION BY FIELD ?

pic:
https://bit.ly/3tWMXqX

**Comment by LonwoLonwo** [comment]
Hello @andreysneg 

Please provide picture one more time, I can't open it.
Or provide me your database name and DBeaver version.

**Comment by andreysneg** [comment]
Where is - Partition by field ?
Version  23.2.3

![WRz9EzQ](https://github.com/dbeaver/dbeaver/assets/14858838/776a8f32-7590-4f11-b383-d483945a0277)

**Reply by LonwoLonwo** [reply]
Partitions Creating is not supported for MySQL/MariaDB yet. Here is the ticket: https://github.com/dbeaver/dbeaver/issues/2627 
You can vote for it.

**Comment by andreysneg** [comment]
How vote ? Where vote ?  In ticket i show only LIKE

**Reply by LonwoLonwo** [reply]
Yes, you can add your like on the main topic.
